src/pool/srv_pool.c

```diff
@@ -8234,6 +8234,74 @@ ds_pool_svc_stop_handler(crt_rpc_t *rpc)
 	pool_svc_stop_handler(rpc, DAOS_POOL_VERSION);
 }
 
+/* TODO: make a real RPC handler ds_pool_rebuild_stop_handler(crt_rpc *rpc) to call this. */
+void
+ds_pool_rebuild_stop(uuid_t pool_uuid, struct rsvc_hint *hint)
+{
+	struct pool_svc *svc;
+	int              rc;
+
+	D_INFO(DF_UUID ": received request to stop rebuild\n", DP_UUID(pool_uuid));
```

Comment:
[Nit] Since the `pool_svc_lookup_leader` call below may return `-DER_NOTLEADER` for 7 to 14 s, would it be more appropriate to use `D_DEBUG` here? If we'd like to log an `INFO` for the stop event, which makes sense to me, then a better place to do so might be somewhere in/during `ds_rebuild_admin_stop`?